gnd/src/main/res/layout/photo_field.xml

```diff
@@ -39,5 +39,14 @@
       android:contentDescription="@string/photo_preview"
       android:scaleType="fitCenter"
       app:imageUri="@{viewModel.uri}" />
+    <ImageButton
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_gravity="top|end"
+      android:onClick="@{() -> viewModel.clearResponse()}"
+      android:visibility="@{viewModel.isEditable ? View.VISIBLE : View.GONE }"
```

Comment:
To keep all conditional logic in the same place (ViewModel), can we instead expose something like `LiveData<Integer> getImageButtonVisibility()` from the view model and refer to it here?